python/tvm/relay/grammar/py3/RelayLexer.tokens

```diff
@@ -56,7 +56,7 @@ CNAME=40
 ':'=16
 'Tensor'=17
 '_'=18
-'v0.0.2'=19
+'v0.0.3'=19
```

Comment:
Can you mark this file ignored in the .gitattributes

Reply:
I had marked them as binary. Is it enough?

Reply:
If this is an auxiliary file, we should delete it from the git and add to gitignore

Reply:
it is deleted.